### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by raulorteg**
* common.plot function now uses tex in plots only if available in the system #32, need to test if eveything is correct once we have data
* adapted the parser now to expect 7 input parameters of the model, since we added DTM parameter the parser needed to be adapted #48 
* added parser method for parsing the continuum file (model.cont) #54

### CODE REVIEW

File: src/parser.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def parse(self, path:str):
         if subdirs["done"]:
```
Comment (eor) [inline src/parser.py:60]: The directory name should probably not be hard-coded. I suggest we change it in a future commit

File: src/parser.py [hunk 2]
```diff
@@ -317,11 +311,46 @@ def parse_emis(self, path:pathlib.PosixPath):
...
+        :param pathlib.PosixPath path: path to the e.g "done/1234" directory
+        """
+        save_path = path.parent.joinpath("cont.pkl")
+        cont_dataframe = pd.read_csv(path, sep="\t")
+        cont_dataframe.to_pickle(save_path)
+        del cont_dataframe
+    
+    def parse_cont(self, path:pathlib.PosixPath):
+        """
+        Given the path to "done" directory
```
Comment (eor) [inline src/parser.py:332]: I think this explanation should also be re-worked in a future commit.

**Review by eor — APPROVED** [review]
Only two minor issues, which can be changed later. Merging the PR, thanks Raul.

File: src/parser.py (hunk 1 above)
Comment (raulorteg) [reply src/parser.py:60]: agreed, probably could be defined in settings as a global variable

Comment (eor) [reply src/parser.py:60]: It already is, see  `common/settings.py`   :-)

**Pull request merged**